docs/shared/cli-install/spm.mdx

```diff
@@ -1,15 +1,13 @@
 The Apollo iOS SPM package includes the Codegen CLI as an executable target. This ensures you always have a valid CLI version for your Apollo iOS version.
 
-To simplify accessing the Codegen CLI, you can run the included `apollo-cli-install` SPM plugin.
+To simplify accessing the Codegen CLI, you can run the included `apollo-cli-install` SPM plugin. This plugin builds the CLI and creates a symbolic link to the executable in your project root.
 
-This plugin builds the CLI and creates a symbolic link to the executable in your project root.
-
-When using a `Package.swift` file, install the CLI by running:
+If using a `Package.swift` file, you can install the CLI by running:
 
 ```bash
 swift package --allow-writing-to-package-directory apollo-cli-install
 ```
 
-After running the installation plugin, a symbolic link to the Codegen CLI named `apollo-ios-cli` is located in your project root folder. You can now run the CLI from the command line with `./apollo-ios-cli`.
+After the plugin installs, it creates a symbolic link to the Codegen CLI (named `apollo-ios-cli`) in your project root folder. You can now run the CLI from the command line using `./apollo-ios-cli`.
 
-> **Note:** Because the `apollo-ios-cli` in your project root is only a symbolic link, it will only work if the compiled CLI exectuable exists. This is generally located in your Xcode Derived Data or the `.build` folder. If these are cleared, you can run the install plugin again to re-build the CLI executable.
+> **Note:** Because the `apollo-ios-cli` in your project root is only a symbolic link, it only works if the compiled CLI executable exists. This is generally located in your Xcode Derived Data or the `.build` folder. If these are cleared, you can run the reinstall plugin to rebuild the CLI executable.
```

Comment:
```suggestion
> **Note:** Because the `apollo-ios-cli` in your project root is only a symbolic link, it only works if the compiled CLI executable exists. This is generally located in your Xcode Derived Data or the `.build` folder. If these are cleared, you can rerun the Install CLI plugin to rebuild the CLI executable.
```